package.json

```diff
@@ -156,7 +156,10 @@
     "@types/uuid": "^9.0.8",
     "randomcolor": "^0.6.2",
     "react-native-aes-crypto": "^2.1.1",
-    "react-native-gifted-chat": "^2.4.0",
-    "react-native-image-picker": "^7.1.2"
+    "react-native-get-random-values": "^1.11.0",
+    "react-native-gifted-chat": "^2.6.3",
+    "react-native-image-picker": "^7.1.2",
+    "react-native-reanimated": "^3.15.3",
+    "react-native-safe-area-context": "^4.11.0"
```

Comment:
Do we need to use this dependency?

Reply:
yes @danhmaisgt , in dependency need this library 
https://github.com/FaridSafi/react-native-gifted-chat